docs/operatorTestPlaybooksCI.md

```diff
@@ -0,0 +1,52 @@
+# Purpose
+This document is to describe about the working of the Upstream Operator Test Playbooks CI Pipeline.
+
+# Introduction
+The Operator Test Playbooks CI Pipeline exists to automatically run tests against pull requests in the redhat-operator-ecosystem/operator-test-playbooks GitHub repository.
+
+# Configuration
+A webhook is configured so that the repository sends events to the github2fedmsg web application.
+Any activity (ex: adding comments) in the repository is therefore sent to github2fedmsg, which then rebroadcasts on the Fedora Messaging bus (fedmsg). 
```

Comment:
I suggest adding: `For more information on github2fedmsg and Fedora Messaging see https://github.com/fedora-infra/github2fedmsg.`